packages/bpk-component-navigation-bar/stories.js

```diff
@@ -107,4 +107,35 @@ storiesOf('bpk-component-navigation-bar', module)
         }
       />
     </div>
+  ))
+  .add('With icon', () => (
+    <div className={getClassNames('bpk-navigation-bar-story')}>
+      <BpkNavigationBar
+        id="test"
+        title={
+          <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
can we get rid of the inline styles and use a class? Also, I think a `span` is semantically more appropriate here.